dev-ml/ocaml-gettext/ocaml-gettext-0.3.7-r1.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit findlib eutils autotools
+
+DESCRIPTION="Provides support for internationalization of OCaml program"
+HOMEPAGE="https://github.com/gildor478/ocaml-gettext"
+SRC_URI="https://github.com/gildor478/ocaml-gettext/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1-with-linking-exception"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc test"
+RESTRICT="!test? ( test )"
+
+PATCHES=( "${FILESDIR}"/ocaml-unsafe-string.patch )
+
+RDEPEND=">=dev-lang/ocaml-3.12.1:=
+	>=dev-ml/ocaml-fileutils-0.4.0:=
+	>=dev-ml/camomile-0.8.3:=
+	sys-devel/gettext
+	dev-ml/camlp4:=
+	"
+DEPEND="${RDEPEND}
+	doc? ( app-text/docbook-xsl-stylesheets dev-libs/libxslt )
+	test? ( dev-ml/ounit )"
```

Comment:
Pop the last " on a new line, and change doc? to have docbook-* and libxslt each on a new line.